repo/packages/A/artifactory/3/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": {
+    "icon-large": "https://raw.githubusercontent.com/JFrogDev/artifactory-dcos/master/images/jfrog_lrg.png",
+    "icon-medium": "https://raw.githubusercontent.com/JFrogDev/artifactory-dcos/master/images/jfrog_med.png",
+    "icon-small": "https://raw.githubusercontent.com/JFrogDev/artifactory-dcos/master/images/jfrog_sml.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "artifactory-pro": "docker.bintray.io/jfrog/artifactory-pro:5.11.1",
```

Comment:
@maheshjfrog As we are using the same image for both. can you combine them to one?